cmd/config-v10.go

```diff
@@ -112,168 +117,191 @@ var serverConfig *serverConfigV10
 
 // GetVersion get current config version.
 func (s serverConfigV10) GetVersion() string {
-	s.rwMutex.RLock()
-	defer s.rwMutex.RUnlock()
+	severConfigMu.Lock()
```

Comment:
This should be a RLock()/RUnlock() here and in other places where members of serverConfigV10 objects are being read.